client/my-sites/importer/importer-header/start-button.jsx

```diff
@@ -56,7 +56,7 @@ class StartButton extends React.PureComponent {
 export default flow(
 	connect(
 		null,
-		{ recordTracksEvent }
+		{ recordTracksEvent, startImport }
```

Comment:
I'm not sure about mapping the flux action to the props here. It seems like it could be misleading or confusing. I'm cool with keeping it though, especially given flux wont be here for much longer, just playing 👿!